src/main/model/Shapes.java

```diff
@@ -1,7 +1,8 @@
-package Abstraction;
+package model;
 
 public class Shapes {
 
 	//TODO redefine toString method to insert the write xml tags for SVG
 	
 }
+
```

Comment:
And I guess their is a blank line in addition, as github does not show the last blank line, here there must be two blank lines right ? (Github usually just shows when their is no empty line at the end, but does not display it.